aten/src/ATen/native/RNN.cpp

```diff
@@ -665,6 +670,15 @@ std::tuple<Tensor, Tensor, Tensor> lstm(
             num_layers, dropout_p, train, bidirectional, batch_first);
     return std::make_tuple(output, hy, cy);
   }
+  if (at::mkldnn_is_acceptable(_input)
+      && (!train || (train && _input.size(2) == hx[0].size(2) && !bidirectional))) {
+    // Disable MKLDNN RNN for training when input_size != hidden_size and bidirectional
+    // since workspace offset is miss calculated on such cases
+    // This issue has been fixed on https://github.com/intel/mkl-dnn since commit d5c4eb5e
```

Comment:
If it's been fixed, can't we simply use that commit?

Reply:
@apaszke 
Current mkl-dnn version under pytorch/third_party/ideep/mkl-dnn still not upgraded after the commit version d5c4eb5e, so the issue is still exist and you can simply use this commit as the workaround for this. Thanks!

Reply:
Sure. My question was if it was possible to update it?